p2p/service/sim.go

```diff
@@ -191,7 +192,7 @@ func (sn *Node) SubscribePeerEvents() (chan crypto.PublicKey, chan crypto.Public
 
 // RegisterProtocol creates and returns a channel for a given protocol.
 func (sn *Node) RegisterProtocol(protocol string) chan Message {
-	c := make(chan Message)
+	c := make(chan Message, config.ConfigValues.BufferSize)
```

Comment:
Please use `DefaultConfig()` instead because we want to get rid of global shared state (`ConfigValues`)